src/components/gridList/gridList.js

```diff
@@ -375,7 +412,8 @@ function GridLayoutFactory($mdUtil) {
       reflow: function(customAnimatorFn) {
         reflowTime = $mdUtil.time(function() {
           var animator = customAnimatorFn || defaultAnimator;
-          tiles.forEach(function(it) {
+          animator(gridStyles.grid.element, gridStyles.grid.style);
+          gridStyles.tiles.forEach(function(it) {
```

Comment:
This approach feeds the animator each tile individually to the animator.
Normally the animator will want the SET of animations and will decide how to animate.

```js
function animator( targets ) {
 targets = angular.isArray(targets) ? targets : [targets];

// First collapse all targets to center
// collapseToCenter(targets);

// Then explode to destination positions
 angular.forEach(function(it) {
   angular.element(it.element).attr(it.styles);
});

} 

}
```